web/src/components/apps/AppVersionHistory.jsx

```diff
@@ -399,6 +405,90 @@ class AppVersionHistory extends Component {
       });
   }
 
+  upgradeAdminConsole = version => {
+    const { app } = this.props;
+
+    this.setState({
+      displayKotsUpdateModal: true,
+      kotsUpdateRunning: true,
+      kotsUpdateStatus: undefined,
+      kotsUpdateMessage: undefined,
+      kotsUpdateError: undefined,
+    });
+
+    fetch(`${process.env.API_ENDPOINT}/app/${app.slug}/sequence/${version.parentSequence}/update-console`, {
+      headers: {
+        "Authorization": Utilities.getToken(),
+        "Content-Type": "application/json",
+      },
+      method: "POST",
+    })
+      .then(async (res) => {
+        if (!res.ok) {
+        const response = await res.json();
+          this.setState({
+            kotsUpdateRunning: false,
+            kotsUpdateStatus: 'failed', // TODO: real value
+            kotsUpdateError: response.error,
+          });
+          return;
+        }
+        this.state.kotsUpdateChecker.start(this.getKotsUpdateStatus, 1000);
+      })
+      .catch((err) => {
+        console.log(err);
+        this.setState({
+          kotsUpdateRunning: false,
+          kotsUpdateStatus: 'failed', // TODO: real value
+          kotsUpdateError: err?.message || "Something went wrong, please try again.",
+        });
+      });
+  }
+
+  getKotsUpdateStatus = () => {
+    const { app } = this.props;
+
+    return new Promise((resolve, reject) => {
+      fetch(`${process.env.API_ENDPOINT}/app/${app.slug}/task/update-admin-console`, {
+      headers: {
+        "Authorization": Utilities.getToken(),
+        "Content-Type": "application/json",
+      },
+      method: "GET",
+    })
+      .then(async (res) => {
+        if (res.status === 404) {
+          // TODO: remove... this is for testing with older kots releases
```

Comment:
cleanup?

Reply:
maybe after next release? otherwise it's impossible to test.  seems harmless.